modules/prebidServerBidAdapter/index.js

```diff
@@ -232,12 +232,17 @@ function doClientSideSyncs(bidders) {
   });
 }
 
-function _getDigiTrustQueryParams() {
-  function getDigiTrustId() {
-    let digiTrustUser = window.DigiTrust && (config.getConfig('digiTrustId') || window.DigiTrust.getUser({member: 'T9QSFKPDN9'}));
+function _getDigiTrustQueryParams(bidRequest = {}) {
+  function getDigiTrustId(bidRequest) {
+    const bidRequestDigitrust = utils.deepAccess(bidRequest, 'bids.0.userId.digitrustid.data');
+    if (bidRequestDigitrust) {
+      return bidRequestDigitrust;
```

Comment:
Let's check the privacy.optout flag and skip if set to true

Reply:
This check is being done in existing code, so I didn't change it.  `getDigiTrustId` is within the `_getDigiTrustQueryParams` function, which does that check before returning data.

Reply:
Here's the line: https://github.com/rubicon-project/Prebid.js/blob/1076efb83eb1eef7248d8f3b2ee65643355e4746/modules/prebidServerBidAdapter/index.js#L247